### ISSUE TITLE
hdf5 1.10.0-patch1: revisions

### ISSUE THREAD

**Issue by iMichka** [Homebrew/homebrew-science#5011]
- [x] abinit (#4934, needs #4938 first)
 - [x] alembic (#5012)
 - [x] alpscore (#5006)
 - [x] anvio (#5055)   
 - [ ] blasr (#4254, failing, waiting on update)
 - [x] blast (#5032)
 - [x] cdo (#5013)
 - [x] cgns (#5008)           
 - [x] cmor (#5045)
 - [x] dealii (#5056)
 - [x] dextractor (#5009)      
 - [x] dynare (#5044)    
 - [x] enblend-enfuse (#5061)     
 - [x] etsf_io (#4938)
 - [x] field3d (#5023)
 - [x] flann (#5014)
 - [x] getdp (#5054)
 - [x] gmt (#5042)
 - [x] gmt4 (#5043)
 - [x] gnudatalanguage (#5015)
 - [x] h5utils (#5010)
 - [x] insighttoolkit (#5060)     
 - [x] kallisto (#5016)
 - [x] libbi (#5041)
 - [x] libminc (#5017)
 - [x] mbsystem (#5040)
 - [x] mlpack (#5039)
 - [x] moose (#5028)
 - [x] moab (#5027)
 - [x] nanopolish (#5029)
 - [x] nco (#5038)
 - [x] nccmp (#5037)
 - [x] ncview (#4949)
 - [x] netcdf (#5007)
 - [x] nexusformat (#5018)
 - [x] nixio (#5019)
 - [x] openimageio (#5053)
 - [ ] openmeeg (broken since Jul. 2015, #4276)
 - [x] octave (#5047)                       
 - [x] petsc (#5024)
 - [x] pcl (#5057)
 - [x] therion (#5058)        
 - [ ] paraview (#5005, #5094, #5082)
 - [x] poretools (#5020)
 - [ ] pbsuite (#5049, blocked by #4254)
 - [x] radx (#5021)
 - [x] seqdb (#5025)
 - [x] silo (#5033)
 - [x] shogun (#5034)
 - [x] sratoolkit (#5022)
 - [x] statismo (#5059)
 - [x] slepc (#5046)
 - [x] sspace-longread (#5048) (was removed, see #5083)
 - [x] swrcfit (#5051)
 - [x] vigra (#5026)
 - [x] vtk (#5050)
 - [x] xmi-msim (#5030)
 - [x] xmgredit (#5034)

**Comment by iMichka** [comment]
Migration done; all the broken formulae were known to be broken before, so nothing new there. A separate issue has been opened for paraview. We can close this now.
